Content.Client/UserInterface/Systems/MenuBar/Widgets/GameTopMenuBar.xaml

```diff
@@ -20,6 +20,7 @@
         BoundKey = "{x:Static ic:EngineKeyFunctions.EscapeMenu}"
         ToolTip="{Loc 'game-hud-open-escape-menu-button-tooltip'}"
         MinSize="70 64"
+        SizeFlagsStretchRatio="70"
```

Comment:
SizeFlagStretchRatio just adds up all of them and then distributes it based on the ratio. Thus, we can simplify 70:42 to 5:3 and it should be the same.
 
(You'll need to fix the rest of them yourself)

```suggestion
        SizeFlagsStretchRatio="5"
```

Reply:
I think it's more obvious where the numbers come from in this case if they're left equal to the X MinSize.

Reply:
Yeah that's fine.